catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -445,6 +463,19 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
```

Comment:
That's probably correct; however, I'm not going to fix it right here. We'll fix that when we move this build plugin out of the submodules and into the root pom.